src/modules/graphics/Font.cpp

```diff
@@ -507,8 +507,12 @@ std::vector<Font::DrawCommand> Font::generateVerticesFormatted(const love::font:
 				auto start = text.cps.begin() + range.getOffset();
 				auto end = start + range.getSize();
 				float numspaces = std::count(start, end, ' ');
+
+				if (text.cps[range.last] == ' ')
```

Comment:
Should this loop backwards until the first non-space character? In case there are multiple spaces in a row

Reply:
I'm not sure if dealing with trailing spaces is worth it in this case, since text is still affected when there's additional spaces in the beginning, or in the middle